app/src/main/java/com/zulip/android/util/Constants.java

```diff
@@ -27,4 +27,5 @@
     //if two continuous messages are from same sender and time difference is less than this then hide it
     public static long HIDE_TIMESTAMP_THRESHOLD = 60 * 1000;// 1 minute
     public static String ACTION_MESSAGE_PUSH_NOTIFICATION = "ACTION_MESSAGE_PUSH_NOTIFICATION";
+    public static String NIGHT_THEME = "isNightTheme";
```

Comment:
Let's follow the convention going on, `public static String NIGHT_THEME = 'NIGHT_THEME'`